FWCore/Integration/test/transformTest_cfg.py

```diff
@@ -27,6 +28,8 @@
 process.maxEvents.input = 4
 
 process.start = cms.EDProducer("IntProducer", ivalue = cms.int32(1))
+if args.exception:
+  process.start = cms.EDProducer("FailingProducer")
```

Comment:
So the test failure is in what is consumed and not in the actual transform call its.

Reply:
Right, basically from anywhere in the data dependence chain of the module registering the transformation call.